frontend/src/common/models/user.ts

```diff
@@ -0,0 +1,5 @@
+export interface IUser {
+  name?: string;
```

Comment:
Better to use the same fields as for backend and name should be required, shouldn't it?

Reply:
I just mocked it up for testing